gedcom/families.py

```diff
@@ -217,3 +220,39 @@ def _validate_death_before_divorce(self, family):
                         family["id"],
                         "NA",
                         msg + wife["id"] + " " + wife["name"])
+
+    def _validate_marrb4div_dates(self, family):
+        """Validate that family marriage date occurs before divorce
+        """
+        if family["divorced_date"] is not None:
+            if family["married_date"] is not None:
+                if family["married_date"] > family["divorce_date"]:
```

Comment:
I see `family["divorced_date"]` and `family["divorced_dated"]` in this function. Which one is it?

Reply:
Updated